Program.cs

```diff
@@ -474,11 +474,19 @@ private static string FindExeFromPath(string ExeName, string ExpectedPathSubstri
 
 		private static ISet<string> RemoveGitFiles(string RootPath, ISet<string> FoundFiles)
 		{
-#if __MonoCS__
-			string git = FindExeFromPath("git");
-#else
-			string git = FindExeFromPath("git.exe");
-#endif
+			string git = "";
+			int pid = (int) Environment.OSVersion.Platform;
+
```

Comment:
Detecting the OS with hard-coded pid values looks fragile. I'm a .NET amateur, so maybe I'm wrong and this is a standard way to detect the OS from .NET. But I found https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.runtimeinformation.isosplatform(v=vs.110).aspx which seems to do things more cleanly.

bool bWindows = System.Runtime.InteropServices.RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
if (!bWindows)
{    //etc...